### DISCUSSION TITLE
Support of reusable workflows

### DISCUSSION THREAD

**Discussion by jguillon** [nektos/act#1092] [Q&A] [answered]
Hi!

One can now define reusable workflows in separate `./.github/workflows/*.yml` files as explained in the [official documentation](https://docs.github.com/en/actions/using-workflows/reusing-workflows). 

It seems that `act` does not support this kind of syntax and only outputs:
```
ERRO[0000] 'runs-on' key not defined in my-workflow-name/my-job-id
```
Am I right?

If so, are you planning on supporting this feature soon?

Best,

@jguillon

**Comment by KnisterPeter** [comment, marked as answer]
Hi @jguillon,
I would love to have act support reusable-workflows. Unfortunately we have very limited resources to implement features and maintain the codebase.
If you are willing to contribute a PR with this would be very welcome.

**Comment by catthehacker** [comment]
Duplicate of #826
